arch/arm/src/samv7/sam_pwm.c

```diff
@@ -957,7 +957,9 @@ static int pwm_start(struct pwm_lowerhalf_s *dev,
                      const struct pwm_info_s *info)
 {
   struct sam_pwm_s *priv = (struct sam_pwm_s *)dev;
+#ifdef CONFIG_PWM_MULTICHAN
   uint32_t regval;
+#endif
 
 #ifdef CONFIG_PWM_MULTICHAN
```

Comment:
```suggestion
```

Reply:
Oh yes, have missed it is right below. Also moved the for loop to correct indention.